docs/se/faulttolerance/01_faulttolerance.adoc

```diff
@@ -0,0 +1,338 @@
+times out can be retried a few times before resorting to a fallback &mdash;assuming
+it never succeeds. Here is how this example would be written using the
+Fault Tolerance API:
+
+[source,java]
+----
+// Create supplier for timeout
+Supplier<? extends CompletionStage<T>> supplierTimeout =
+    () -> Timeout.create(Duration.ofMillis(10)).invoke(this::mayTakeVeryLong);
+
+// Create supplier for retry using timeout's
+Retry retry = Retry.builder()
+                   .retryPolicy(Retry.JitterRetryPolicy.builder()
+                                     .calls(3)
+                                     .delay(Duration.ofMillis(100))
+                                     .build())
+                   .build();
+Supplier<? extends CompletionStage<T>> supplierRetry = () -> retry.invoke(supplierTimeout);
+
+// Finally invoke this::mayTakeVeryLong wrapping supplier for retry using fallback
+Single<T> single = Fallback.create(
+    throwable -> Single.just(lastKnownValue)).invoke(supplierRetry);
+----
+
+The exact manner in which handlers are combined depends on the use case,
+but generally the order starting from innermost to outermost should be: bulkhead,
+timeout, circuit breaker, retry and fallback. That is, bulkhead is the first
+handler executed and fallback the last one.
+
+NOTE: This is the ordering used by the MicroProfile Fault Tolerance implementation
+in Helidon when a method is decorated with multiple annotations.
+
+=== Revisiting Multi's
+
+All the examples presented so far have focused on invocations returning
+a single value of type `Single<T>`. If the invocation in question can return
+more than one value (i.e., a `Multi<T>`) then all that is needed is to use
+the method `invokeMulti` instead of `invoke`. The supplier passed to this
+method must return a `Flow.Publisher<T>` instead of a `CompletionStage<T>`.
+
+A `Flow.Publisher<T>` is a generalization of a `Single<T>` that can
+produce zero or more values. Note that a `Flow.Publisher<T>`, unlike a
+`Single<T>`, can report an error after
+producing one or more values, introducing additional challenges if all
+values must be processed transactionally, that is, in an all or nothing
+manner.
+
+The following example creates an instance of `Retry` and invokes
+the `invokeMulti` method, it then registers a subscriber to process
+the results:
+
+[source,java]
+----
+Retry retry = Retry.builder()
+                   .retryPolicy(Retry.JitterRetryPolicy.builder()
+                                     .calls(2)
+                                     .build())
+                   .build();
+Multi<Integer> multi = retry.invokeMulti(() -> Multi.just(0, 1, 2));
+
+IntSubscriber ts = new IntSubscriber();
+multi.subscribe(ts);
+ts.request(Integer.MAX_VALUE);
+----
+
+The call to `Multi.just(0, 1, 2)` simply returns a multi that produces
+the integers 0, 1 and 2. If an error was generated during this process,
+the policy will retry the call one more time &mdash;for a total of 2
```

Comment:
does the em dash render correctly?

Reply:
Actually, using the view option here on Github, it renders fine.